pkgs/applications/editors/pulsar/001-patch-wrapper.patch

```diff
@@ -0,0 +1,28 @@
+--- ./resources/pulsar.sh	2023-03-16 04:11:14.000000000 +0100
++++ ./resources/pulsar.sh	2023-03-24 14:37:13.468813964 +0100
+@@ -123,22 +123,10 @@
+ elif [ $OS == 'Linux' ]; then
+   SCRIPT=$(readlink -f "$0")
+ 
+-  PULSAR_PATH="/opt/Pulsar/pulsar"
++  PULSAR_DEFAULT="/opt/Pulsar/pulsar"
```

Comment:
We are already patching this and know this path is wrong, so we should just remove it